ResearchKit/Common/ORKFormStep.h

```diff
@@ -129,6 +129,14 @@ ORK_CLASS_AVAILABLE
 @property (nonatomic, copy, readonly) NSString *identifier;
 
 /**
+ A Boolean value indicating whether the form item is optional.
+ 
+ The default value of this property is `NO`. When the value is `YES`, this form item doesn't need
```

Comment:
Shall we make default value to be `YES`? 
To align with `ORKStep`'s `optional`.